app/models/person.rb

```diff
@@ -1,17 +1,19 @@
 class Person < ActiveRecord::Base
   #code to get next un-used id # in the 800 range
   #((801...900).to_a - (array_from_db_of_taken_numbers))[0]
-  before_save :title_order
+  attr_accessible :firstname, :lastname, :gender, :date_of_birth, :zipcode, :city, :state, :status, :end_date, :department, :icsid, :title, :title_ids, :start_date, :division1, :division2, :comments
+  attr_accessible :channels_attributes
 
-  attr_accessible :firstname, :lastname, :status, :icsid, :department, :city, :state, :zipcode, :start_date, :end_date , :title, :gender, :date_of_birth,:division1, :division2, :channels_attributes, :title_ids, :title_order, :comments
```

Comment:
@prestoncopeland My specific question: Was there a reason you removed the `title_order` attr_accessor?  Was it interfering with the `before_save :title_order` call?